README.md

```diff
@@ -98,28 +98,30 @@ parameterValidator.validateAsync(options, [ 'firstName', 'lastName' ])
 ### Parameters for `validate` and `validateAsync`
 
 ```
-param:   {Object}  paramsProvided      - The names and values of provided parameters
```

Comment:
I added a doc for the new `errorClass` option and updated the spacing for the other params.